### PR TITLE
See #17

### PR DESCRIPTION
**Pull request by vimalaguti**
- removed javacpp dep
- do not remove last version number
- update sbt and scala

### CODE REVIEW

File: src/main/scala/org/bytedeco/sbt/javacpp/Plugin.scala [hunk 1]
```diff
@@ -37,25 +37,15 @@ object Plugin extends AutoPlugin {
   private def javaCppPresetDependencies: Def.Setting[Seq[ModuleID]] = {
     import autoImport._
     libraryDependencies ++= {
-      val majorMinorJavaCppVersion = majorMinorOnly(javaCppVersion.value)
       javaCppPresetLibs.value.flatMap {
-        case (libName, libVersion) => {
-          val generic = "org.bytedeco.javacpp-presets" % libName % s"$libVersion-$majorMinorJavaCppVersion" classifier ""
+        case (libName, libVersion) =>
+          val generic = "org.bytedeco.javacpp-presets" % libName % s"$libVersion-${javaCppVersion.value}" classifier ""
```
Comment (lloydmeta) [inline src/main/scala/org/bytedeco/sbt/javacpp/Plugin.scala]: I think we need a smarter check here for backwards compatibility; to take into account the fact most of the existing versions depend on *not* having the patch version (see https://github.com/bytedeco/sbt-javacpp/issues/17#issuecomment-448978403)

**Review by lloydmeta — COMMENTED** [review]
Thanks for putting the time in to make this happen :)

Given that the build is failing, and I don't really want to spend too much time figuring out why, my advice is to undo the sbt build-related changes and focus getting the versioning fix right.

**Comment by vimalaguti** [comment]
I should have fixed

**Review by lloydmeta — APPROVED** [review]
LGTM.

**Comment by lloydmeta** [comment]
Thanks for your great work !

**Pull request merged**